tox.ini

```diff
@@ -117,8 +117,7 @@ usedevelop=true
 skip_install = true
 usedevelop = false
 deps =
-    # Old automat version for Twisted
-    Automat == 0.3.0
+    Automat == 0.8.0
```

Comment:
Also a bit surprised by this one. I would have thought that twisted would specify which version of `Automat` it needs!